### DISCUSSION TITLE
I am running akhq 0.18.0 but need help on configure health endpoint

### DISCUSSION THREAD

**Discussion by ankurjains** [tchiotludo/akhq#855] [Q&A]
I am running akhq in kubernetes with 8080 port exposed and for akhq. But documentation says 28081 for health endpoint which i can not access when i hit localhost:28081 as akhq is running on 8080. if i change port to 8080 in endpoint section from 28081 container wont start and throw error
Unable to start server. Additional specified server port 8080 already in use.
2021-10-14 12:41:06,609 ERROR main       i.m.runtime.Micronaut      Error starting Micronaut server: Unable to start Micronaut server on port: 8080
io.micronaut.http.server.exceptions.ServerStartupException: Unable to start Micronaut server on port: 8080

Could anyone help on configuring health endpoint so that i can use it in k8s liveness test. lack java skills

**Comment by ankurjains** [comment]
@tchiotludo

**Comment by tchiotludo** [comment]
You can't change the endpoint port to the same port of the app.
Since default application have an endpoint configuration, you can't change it to the same port. 

Whenever, it's a bad practice & security issues to expose health on the same port.
So the endpoint is not reachable outside of the pod and it's a good things. 

But I wonder why you have some much difficulty, the official helmcharts have all the configuration required to have a running service on k8s with healthcheck.

**Reply by ankurjains** [reply]
We in our org are not supporting helm but I have created my own config and yamls for k8s deployment. For that I need to configure health endpoint but when i hit from pod localhost:28081/health nothing comes up

**Reply by tchiotludo** [reply]
are you using the last version ?

**Reply by ankurjains** [reply]
yes i m using 18.0 version
